### PR TITLE
Wrote 15 unit tests for 9 models using `django.test` package, Created an API endpoint for fetching a unique comment for a particular article

### PR DESCRIPTION
**Pull request by dinakar17**
unittests

One Unit Test failure suggests that the expected exception is not being raised when trying to create another admin for the same community.

Created an API endpoint for fetching one comment (review) about a specific article. The objective is to use this to create a shareable URL link for each comment for SEO purposes.
image

### CODE REVIEW

File: app/views.py [hunk 1]
```diff
@@ -901,6 +901,19 @@ def destroy(self, request, pk ):
         super(ArticleViewset, self).destroy(request,pk=pk)
 
         return Response(data={"success": "Article successfully deleted"})
+    
+    @action(methods=['get'], detail=True, url_path='comment/(?P<comment_id>.+)', permission_classes=[ArticlePermission])
```
Comment (JyothiSwaroopReddy07) [inline app/views.py:905]: This function is not needed for now. As I have implemented the already implemented. Please dont include this function

File: app/tests/test_models.py [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+        self.assertEqual(community_member.community, self.community)
+        self.assertEqual(community_member.user, self.user)
+        self.assertFalse(community_member.is_reviewer)
+        self.assertFalse(community_member.is_moderator)
+        self.assertTrue(community_member.is_admin)
+
+    def test_community_member_str(self):
+        self.assertEqual(str(self.community_member), f'{self.user} - {self.community}')
+
+    def test_unique_admin_per_community(self):
```
Comment (JyothiSwaroopReddy07) [inline app/tests/test_models.py:194]: Each community can have more than one admin. Please remove the intented exception error. The testcase works then

**Review by JyothiSwaroopReddy07 — CHANGES_REQUESTED** [review]

**Comment by sushmanthreddy** [comment]
@dinakar17 can u remove all cache files?

use .gitignore to remove them ,pls remove untracked files..

**Comment by JyothiSwaroopReddy07** [comment]
Can also make a git pull to add some changes to which are already present in backend

**Comment by dinakar17** [comment]
I've synced both the main branches of the projects.

image

**Review by JyothiSwaroopReddy07 — APPROVED** [review]

**Comment by JyothiSwaroopReddy07** [comment]
done merged it. thanks for your work

**Pull request merged**